### ISSUE TITLE
Support TOML 1.0

### ISSUE THREAD

**Issue by apirogov** [gscho/toml-lint#3]
This is valid TOML, but your toml-lint website says it isn't:

```toml
[tool.poetry]
name = "dummy-project"
version = "0.1.0"
description = ""
authors = ["John Doe <j.doe@example.com>"]
readme = "README.md"
packages = [{include = "dummy_project"}]

include = [
  # having both a string and an object here seems to trigger the problem:
  "CHANGELOG.md",
  { path = "tests", format = "sdist" },
  # ----
]

[tool.poetry.dependencies]
python = "^3.8"
pyproject-parser = "^0.8.0"
tomli = "^2.0.1"


[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"
```

Your site comes quite at the top when searching for "check TOML", and it was e.g. used to [quickly dismiss an issue](https://github.com/proycon/codemetapy/issues/40).

Would be nice if you update it to support TOML 1.0, as being the first TOML checker tool google presents makes you somewhat of an "authority".

If this is too much work, maybe at least state for which TOML spec version your site is. 

Thanks!

**Comment by gscho** [comment]
@apirogov the site has the spec version listed in the top left hand corner. 

I do plan to update the parsing library to support spec v1.0. Unfortunately the current library this site uses will need to be swapped out completely for another since it is never going to support toml v1.0

**Comment by gscho** [comment]
Closed by #4
